<launch>
	<node pkg="laser3d" name="point_cloud_converter" type="bin_point_cloud_converter_node" >
		<remap from="points_in" to="cloud_timestamp"/>
		<remap from="points2_out" to="cloud_timestamp_2" />
	</node>
</launch>
